processes.go

```diff
@@ -63,6 +63,14 @@ type Process struct {
 	Memory   constraints.Memory   `json:"Memory,omitempty"`
 	CPUShare constraints.CPUShare `json:"CPUShare,omitempty"`
 	Nproc    constraints.Nproc    `json:"Nproc,omitempty"`
+	Expose   *Exposure            `json:"Expose,omitempty"`
+}
+
+// Exposure holds exposure settings for the Process.
+type Exposure struct {
+	Protocol string `json:"Protocol"`
+	External bool   `json:"External"`
```

Comment:
Is it worth maybe making this a string, rather than a bool?  I could see a world where we have different 'types' of subnets, and where the ELB goes is based on this.  Maybe not worth dealing with here.